Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -761,15 +766,15 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     }
     
     private func registerDocuments(from bundle: DocumentationBundle) throws -> (
-        technologies: [SemanticResult<Technology>],
+        tutorialTableOfContents: [SemanticResult<TutorialTableOfContents>],
```

Comment:
Same here:


```
tutorialTableOfContentsCollection: [SemanticResult<TutorialTableOfContents>]
```

Reply:
Added "Results" suffix in https://github.com/swiftlang/swift-docc/pull/1039/commits/62de8fe3c83fb7081db1ef5cdc4f7cde7ad00a7c